Sources/JXKit/JXConvertible.swift

```diff
@@ -121,12 +96,160 @@ extension Array: JXConvertible where Element: JXConvertible {
     }
 }
 
-extension JXValue {
-    /// Sets a `JXConvertible` in this value object.
-    /// - Parameters:
-    ///   - key: The key to set.
-    ///   - object: The `JXConvertible` to convert.
-    public func set<T: JXConvertible>(_ key: String, convertible object: T) throws {
```

Comment:
This just seemed odd to have, given that all other APIs require first converting values to JXValues.